config/locales/en.yml

```diff
@@ -29,5 +29,18 @@
 # To learn more, please read the Rails Internationalization guide
 # available at https://guides.rubyonrails.org/i18n.html.
 
+
+
```

Comment:
because you are setting to use 2 languages english and japanese => you should define another file `ja.yml` in `config/locales/` folder